vault_service/tests/test_bumblebee.py

```diff
@@ -15,10 +16,24 @@
 class TestSite(TestCase):
     '''Tests that each route is an http response'''
 
+    postgresql_url_dict = {
```

Comment:
Not critical but it looks like all the tests have the same duplicated attributes and methods, maybe we could create a VaultTestCase class (which will inherit from TestCase) with all the common code and make all the tests inherit VaultTestCase instead of TestCase.